Neo4jClient/GraphClientFactory.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Neo4jClient
+{
+    public class GraphClientFactory : IGraphClientFactory
+    {
+        private readonly NeoServerConfiguration _configuration;
+
+        public GraphClientFactory(NeoServerConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public IGraphClient Create()
+        {
+            var client = new GraphClient(
```

Comment:
If `configuration` is null - this will throw an exception - should we be checking for the value?